cibyl/plugins/openstack/glance.py

```diff
@@ -0,0 +1,48 @@
+"""
+#    Copyright 2022 Red Hat
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+"""
+from typing import Optional
+
+from cibyl.cli.argument import Argument
+from cibyl.models.model import Model
+
+
+class Glance(Model):
```

Comment:
I'm thinking that maybe we should keep Glance properties as part of the already existing `Storage` model. I think that there won't be too many properties aside from backend in it, and having separated Glance, Manila models would be overkill

Reply:
Ok

Reply:
I have made the changes, take a look.